includes/unenroll.php

```diff
@@ -232,6 +249,16 @@ function unenrollstu($cid,$tounenroll,$delforum=false,$deloffline=false,$withwit
 
 		$query = "DELETE FROM imas_content_track WHERE userid IN ($stulist) AND courseid=$cid";
 		$DBH->query($query); //values already sanitized
+
+		$result = TeacherAuditLog::addTracking(
+			$cid,
+			"Unenroll",
+			null,
+			array(
+				"unenrolled"=>$stulist,
+				"grades"=>$grades

```

Comment:
Do we also want to capture and log any offline or forum grades recorded in imas_grades?